packages/fxa-auth-server/lib/oauth/routes/index.js

```diff
@@ -113,6 +113,7 @@ exports.clients = [
 ];
 
 routes.forEach(function(route) {
+  route.config.cors = { origin: 'ignore' };
```

Comment:
nit: maybe for a followup, but it might be worth checking for any existing `cors` declaration on the route so that we don't mysteriously overwrite someone's future changes to these routes.